### PR TITLE
[6.5.x] RHBPMS-4638 - SQLGrammarException when retrieving user tasks

### PR DESCRIPTION
**Pull request by mswiderski**
@sutaakar do you mind verifying this on 6.5 bits before merging?

### CODE REVIEW

**Comment by mswiderski** [comment]
jenkins retest this

File: jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/CommonUtils.java [hunk 1]
```diff
@@ -42,4 +42,16 @@ public static Long getProcessInstanceId(Command<?> command) {
         return null;
 		
 	}
+	
+	// to compensate https://hibernate.atlassian.net/browse/HHH-8091 add empty element to the roles in case it's empty
+	public static List<String> getAuthenticatedUserRoles(IdentityProvider identityProvider) {
+        List<String> roles = identityProvider != null ? identityProvider.getRoles() : new ArrayList<>();
```
Comment (sutaakar) [inline jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/CommonUtils.java]: Seems to be a compilation issue here, new ArrayList<>() should be new ArrayList<String>()

**Comment by mswiderski** [comment]
should be fixed now. not sure why I got jenkins report saying there is only test failure... or I read it wrong. thanks @sutaakar

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by sutaakar** [comment]
@mswiderski Yes, it fix the issue.

**Pull request merged**